public/app/percona/shared/services/user/User.types.ts

```diff
@@ -5,8 +5,10 @@ export interface UserStatusResponse {
 export interface UserDetailsResponse {
   user_id: number;
   product_tour_completed?: boolean;
+  alerting_tour_completed?: boolean;
```

Comment:
We don't need to overengineer this now, but what happens if we have 10 tours in the future? Most likely we can't have 10 keys on the API, user state and so on, probably at that time we need to encapsulate this better.

Reply:
I agree, I think we need to rethink this with the new Onboarding as there will be a possibility to view/dismiss each feature tour/etc. at any time in the "Coach".

Reply:
👍